Dockerfile-dev

```diff
@@ -3,15 +3,19 @@
 
 FROM openjdk:8-jdk
 ENV JENKINS_UC https://updates.jenkins.io
+ENV CASC_JENKINS_CONFIG file:///casc/jenkins.yaml
```

Comment:
It would be nice to point this to a directory, so that the configuration can be split into multiple files?

Reply:
I agree that it should be a directory by default. If it does not work, it might be a bug in JCasC.

```suggestion
ENV CASC_JENKINS_CONFIG /casc
```

Reply:
The jenkins helm chart relies on the directory behaviour: https://github.com/helm/charts/tree/master/stable/jenkins

Reply:
Updating the JCasC plugin from 0.30 to 0.32 solved the issue. This change is in the bom as well, but not yet released, so we'll need to override it until a new version is released (changed here: https://github.com/jenkinsci/bom/commit/64c6b7dd6c51d096833fcd01f04fc2551fa20f11)

Reply:
Overriding BOM is fine for now